script/elevate-cpanel.PL

```diff
@@ -1545,6 +1673,12 @@ sub post_upgrade_check ($self) {
     return 0;
 }
 
+=pod
+    > cat /var/log/leapp/answerfile
```

Comment:
this answer file should not be modifed if no_leapp of course.

Reply:
this is not happening under `--no-leapp` the function is not called